robo4j-socket-http/src/main/java/com/robo4j/socket/http/json/JsonDocument.java

```diff
@@ -34,7 +34,8 @@ public final class JsonDocument {
 	private final Map<String, Object> map = new LinkedHashMap<>();
 	private final Type type;
 
-	JsonDocument(Type type) {
+	// TODO: review the public constructor caused by JPMS
```

Comment:
Remove comment or document (at least one public constructor should be available here anyways)

Reply:
I'll create ticket, about Json parser as there are more challenges